panoptes_client/project_preferences.py

```diff
@@ -89,4 +89,48 @@ def save_settings(cls, project=None, user=None, settings=None):
         else:
             raise TypeError
 
+    @classmethod
+    def fetch_settings(cls, project=None, user=None):
+        """
+        Fetch project preference settings for a particular project and user(optional).
+
+        - **user** and **project** can be either a :py:class:`.User` and
+          :py:class:`.Project` instance respectively, or they can be given as
+          IDs.
+        - **user** parameter is optional and only **project** is required
+
+        Examples::
+
+            ProjectPreferences.fetch_settings(User(1234), Project(1234))
+            ProjectPreferences.fetch_settings(1234, 1234)
+            ProjectPreferences.fetch_settings(User(1234))
+            ProjectPreferences.fetch_settings(1234)
+        """
+
+        _user_id = None
+        _project_id = None
+
+        if isinstance(project, Project):
+            _project_id = project.id
+        elif isinstance(project, (int, str,)):
+            _project_id = project
+        else:
+            raise TypeError
+
+        if isinstance(user, User):
+            _user_id = user.id
+        elif isinstance(user, (int, str,)):
+            _user_id = user
+
+        params = {'user_id': _user_id, 'admin': True}
+
+        if _project_id is not None:
```

Comment:
This check for _project_id feels redundant since you have a check that raises TypeError that sets the project id.